SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -756,18 +740,12 @@ void ActorSystem::AttachDynamicSubobject(AActor* Actor, Worker_EntityId EntityId
 
 	Channel->CreateSubObjects.Add(Subobject);
 
-	ForAllSchemaComponentTypes([&](ESchemaComponentType Type) {
-		Worker_ComponentId ComponentId = Info.SchemaComponents[Type];
-
-		if (ComponentId == SpatialConstants::INVALID_COMPONENT_ID)
-		{
-			return;
-		}
-
+	for (auto ComponentId : *PendingDynamicSubobjectComponents.Find(EntityId))
```

Comment:
Can PendingDynamicSubobjectComponents ever not contain EntityId here?

Reply:
No it's not possible - or if it is then we're resolving a dynamic component that has no matching spatialos components.